package.json

```diff
@@ -30,16 +30,16 @@
   },
   "homepage": "https://github.com/mixmaxhq/eslint-config-mixmax#readme",
   "optionalDependencies": {
-    "babel-eslint": "^8.2.1",
-    "eslint-plugin-flowtype": "^2.41.0",
-    "eslint-plugin-react": "^7.1.0"
+    "babel-eslint": "^10.0.2",
+    "eslint-plugin-flowtype": "^3.11.1",
+    "eslint-plugin-react": "^7.14.2"
   },
   "peerDependencies": {
-    "eslint": ">=5.6.1"
+    "eslint": ">=6.0.1"
```

Comment:
Do you think we even need to specify this as a peer dependency? I guess it doesn't hurt :man_shrugging:

Reply:
My thinking is that it makes npm yell at us so we can safely use 6.0.1 features without having to document the min eslint version 🤷‍♂